packages/common/src/ui/components/Banner/Banner.tsx

```diff
@@ -53,11 +68,22 @@ export const Banner = ({
         </div>
       )}
       <div
-        className={cn(styles.descriptionContainer, { [descriptionClassName]: descriptionClassName })}
+        className={cn(styles.contentContainer, { [descriptionClassName]: descriptionClassName })}
         data-testid="banner-description"
       >
-        <Text className={styles.message}>{message}</Text>
-        {description && <div>{descriptionElement}</div>}
+        <div className={cn(styles.descriptionContainer)}>
+          <Text className={styles.message}>{message}</Text>
+          {withLink && (
+            <>
+              <Link to="">{linkMessage}</Link>
+              <Text className={styles.message}>{messagePartTwo}</Text>
+            </>
+          )}
```

Comment:
Do you think we can do without `withLink`? Maybe something like this?

```suggestion
          <>
              {linkMessage && <Link to="">{linkMessage}</Link>}
              {messagePartTwo && <Text className={styles.message}>{messagePartTwo}</Text>}
          </>
```

Reply:
Yeah, you're right, I can just reduce the props.